README-ja_JP.md

```diff
@@ -35,10 +35,8 @@ Eureka は Tampermonkey/GreasyMonkey に対応するユーザースクリプト
 - [x] Creaticode
 - [x] Adacraft
 - [x] PenguinMod
-- [x] ElectraMod *
-- [x] XPLab *
-
-*\*: CI ビルドだけ使用できます*
```

Comment:
We do not consider the situation where other editors actively introduce Charlotte, and we even recommend that in this case you actively disable Charlotte in the script manager to avoid potential conflicts.

Reply:
ElectraMod and XPlab are already supported on [FurryR's commit](https://github.com/EurekaScratch/eureka/commit/7df3086759a48981e4cb5a42cde8128fb17ac3e3), i just added new TurboWarp CN editor and removed the * from XPlab and ElectraMod beause they're not only available on ci builds.